digdag-docs/src/operators/td.md

```diff
@@ -9,6 +9,14 @@
     +simple_query:
       td>: queries/simple_query.sql
 
+    +simple_query_expanded:
+      td>:
+      data: "SELECT '${session_id}' FROM nasdaq"
+
```

Comment:
I tried to do this **data** parameter with v0.9.39. The error happened as follows.
```
2019-08-09 19:09:20 +0900 [ERROR] (0041@[0:yy_td]+test1+task1): Configuration error at task +test1+task1: Parameter 'query' is required but not set (config)

```

Reply:
`data:` needs an indent and the difference between `query:` was wrong. I updated the description but how to express the difference is little difficult...

Reply:
I confirmed it. Thanks!